xapi-profiles-communication.md

```diff
@@ -323,17 +375,43 @@ partial | non empty            | outcome could be interpreted as success with no
 
 
-A Pattern only matches if it matches greedily. That is, each optional, zeroOrMore, oneOrMore, and alternate Pattern MUST always be considered to match the maximum length possible before considering any Patterns later in a sequence. No backtracking is allowed. This constrains useful Patterns, but guarantees efficient processing, as once a Statement is matched it does not need to be reconsidered (except in cases where it is part of an ultimately unmatched alternate).
-When checking previously collected Statements for matching a Pattern, ordering MUST be based on timestamp. In the event two or more Statements have identical timestamps, any order within those Statements is allowed.
-When checking Statements for matching a Pattern upon receipt, ordering MUST be based on receipt order insofar as that can be determined. If Statements are received in the same batch and they are being checked upon receipt, within the batch Statements MUST be ordered first by timestamp, and if timestamps are the same, by order within the Statement array, with lower indices earlier.
+A Pattern only matches if it matches greedily. That is, each optional, zeroOrMore, oneOrMore, 
+and alternate Pattern MUST always be considered to match the maximum length possible before 
+considering any Patterns later in a sequence. No backtracking is allowed. This constrains 
+useful Patterns, but guarantees efficient processing, as once a Statement is matched it does 
+not need to be reconsidered (except in cases where it is part of an ultimately unmatched 
+alternate).
 
+When checking previously collected Statements for matching a Pattern, ordering MUST be based 
+on timestamp. In the event two or more Statements have identical timestamps, any order within 
+those Statements is allowed.
+
+When checking Statements for matching a Pattern upon receipt, ordering MUST be based on receipt 
+order insofar as that can be determined. If Statements are received in the same batch and they 
+are being checked upon receipt, within the batch Statements MUST be ordered first by timestamp, 
+and if timestamps are the same, by order within the Statement array, with lower indices earlier.
 
 ## <a name="3.0">3.0</a> Libraries
 
-Any library that implements the algorithms given here, exposing all of the listed functions, will be an xAPI Profile Processor library.
+Any programming library that implements the algorithms given here, exposing all of the listed 
+functions, will be an xAPI Profile Processor library.
+
+For each of the `validates`, and `follows`, the Profile Server will provide web APIs that are 
+strongly Not Recommended for production usage, but are suitable for experimentation and 
+demonstration. It is against best practice to go out to a third party server (or any server, 
```

Comment:
I mean, the sentence that starts here is true, but anyone technical reading this is going to find it very odd to 1) include something phrased like this and 2) include something so obvious to anyone developing such systems